tests/conftest.py

```diff
@@ -40,6 +39,14 @@ def pytest_configure():
         PASSWORD_HASHERS=(
             'django.contrib.auth.hashers.MD5PasswordHasher',
         ),
+        **{
```

Comment:
1. Only way I could think to handle `MIDDLEWARE` vs `MIDDLEWARE_CLASSES`.  Is there a better way of doing this?